packages/app/components/misc/interact/MintButton.tsx

```diff
@@ -1,29 +1,23 @@
 'use client'
-import { useContext, useEffect, useState } from 'react'
-import { useAccount } from 'wagmi'
-// import CastrABI from '@/lib/contracts/VideoFactoryAbi'
-import { Button } from '@/components/ui/button'
-import { ConnectKitButton } from 'connectkit'
-import { ModalContext } from '@/lib/context/ModalContext'
-import { base } from 'viem/chains'
+
 import Image from 'next/image'
```

Comment:
You are free to delete the MintButton component, we won't be using it any longer.